cmd_search.c

```diff
@@ -58,7 +58,9 @@ int cmd_search(context_t *context) {
       "--title         DEPRECATED. Same as --name.\n"
       "--all           Require all conditions match a window. Default is --any\n"
       "--any           Windows matching any condition will be reported\n"
-      "--sync          Wait until a search result is found.\n"
+      "--sync [T [A]]  Wait T * A seconds until a search result is found.\n"
+      "                T - sync waiting time till next attempt, default: 0.5\n"
```

Comment:
I think I'd prefer keeping --sync a no_argument and maybe instead providing this functionality with --sync-timeout? 

Additionally, I think having users say "check every `T` seconds" feels like an unnecessary configuration detail -- I think we can choose the right check interval ourselves in the code and have users provide how long they want to wait for (`--sync --sync-timeout 5` for 5 second timeout) ?

Alternately, I am open to being convinced that `--sync [optional_timeout]` is better than two separate flags.